coeus-impl/src/main/resources/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetSettings.xml

```diff
@@ -30,7 +30,7 @@
 				<property name="items">
 					<list>
 						<bean parent="Uif-PrimaryActionButton" p:actionLabel="Apply Changes" p:methodToCall="saveBudgetSettings"
-							p:order="10" p:refreshId = "PropBudget-BudgetSettings-Dialog" p:dialogDismissOption=""
+							p:order="10" p:dialogDismissOption="REQUEST"
```

Comment:
The issue is that then the page doesn't refresh when the dialog is dismissed.  What do you suggest that we do?